app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/checkemail/CheckEmailScreen.kt

```diff
@@ -102,105 +95,119 @@ fun CheckEmailScreen(
-            ) {
-                val goBackAnnotatedString = createAnnotatedString(
-                    mainString = stringResource(
-                        id = R.string.no_email_go_back_to_edit_your_email_address,
-                    ),
-                    highlights = listOf(stringResource(id = R.string.go_back)),
-                    tag = TAG_URL,
-                )
-                ClickableText(
-                    text = goBackAnnotatedString,
-                    onClick = {
-                        goBackAnnotatedString
-                            .getStringAnnotations(TAG_URL, it, it)
-                            .firstOrNull()?.let {
-                                viewModel.trySendAction(CheckEmailAction.CloseClick)
-                            }
-                    },
-                )
-                Spacer(modifier = Modifier.height(32.dp))
-                val logInAnnotatedString = createAnnotatedString(
-                    mainString = stringResource(
-                        id = R.string.or_log_in_you_may_already_have_an_account,
-                    ),
-                    highlights = listOf(stringResource(id = R.string.log_in)),
-                    tag = TAG_URL,
-                )
-                ClickableText(
-                    text = logInAnnotatedString,
-                    onClick = {
-                        logInAnnotatedString
-                            .getStringAnnotations(TAG_URL, it, it)
-                            .firstOrNull()?.let {
-                                viewModel.trySendAction(CheckEmailAction.LoginClick)
-                            }
-                    },
-                )
-            }
             Spacer(modifier = Modifier.navigationBarsPadding())
         }
     }
 }
+
+@Suppress("LongMethod")
+@Composable
+private fun CheckEmailContent(
+    email: String,
+    onOpenEmailAppClick: () -> Unit,
+    onChangeEmailClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Spacer(modifier = Modifier.height(32.dp))
+        Image(
+            painter = rememberVectorPainter(id = R.drawable.open_email),
+            contentDescription = null,
+            contentScale = ContentScale.Fit,
+            modifier = Modifier
+                .size(100.dp),
+        )
+        Spacer(modifier = Modifier.height(32.dp))
+        Text(
+            text = stringResource(id = R.string.check_your_email),
+            textAlign = TextAlign.Center,
+            style = MaterialTheme.typography.titleMedium,
+            color = MaterialTheme.colorScheme.onSurface,
+            modifier = Modifier
+                .padding(horizontal = 8.dp)
+                .wrapContentHeight()
+                .fillMaxWidth(),
+        )
+        Spacer(modifier = Modifier.height(8.dp))
+
+        @Suppress("MaxLineLength")
```

Comment:
I don't think we need this

Reply:
detekt fails w/o when I run it.

Reply:
Weird

Reply:
oh no its actually just a PEBKAC, (aka me) yeah this one is not needed for some reason I thought this was on line 175 suppress tag.